search/intermediate_result.cpp

```diff
@@ -255,11 +256,14 @@ void FillDetails(FeatureType & ft, Result::Details & details)
   if (details.m_isInitialized)
     return;
 
-  details.m_airportIata = ft.GetMetadata(feature::Metadata::FMD_AIRPORT_IATA);
-  details.m_brand = ft.GetMetadata(feature::Metadata::FMD_BRAND);
-
+  auto const airportIata = std::string(ft.GetMetadata(feature::Metadata::FMD_AIRPORT_IATA));
+  
+  auto brand = std::string(ft.GetMetadata(feature::Metadata::FMD_BRAND));
```

Comment:
I find the syntax a bit odd, in my opinion.

```cpp
std::string const airportIata = ft.GetMetadata(feature::Metadata::FMD_AIRPORT_IATA);
std::string brand = ft.GetMetadata(feature::Metadata::FMD_BRAND);
```

Reply:
ft.GetMetadata returns a std::string_view that has to be converted to std::string

Reply:
`std::string brand {ft.GetMetadata(feature::Metadata::FMD_BRAND)};` ?

Reply:
> `std::string brand {ft.GetMetadata(feature::Metadata::FMD_BRAND)};` ?

Changed